### ISSUE TITLE
Celebrating best student organization 2023

### ISSUE THREAD

**Issue by skivdal** [fribyte-code/fribyte.no#70]
I think we should do something special with our frontpage for a little while celebrating this!

I'm thinking something flashy, with a thank you and congratulations to all members, along with something promoting us for potential new members and customers :)

@nheek @simsine @SigvardHelsingen @Erik-G-I anyone interested in designing this?

**Comment by nheek** [comment]
I'm up for it. But will you provide a text or design mockup too? Also, I assume this is asap or?

**Comment by polsevev** [comment]
Yes it should be like a banner or smthn Or maybe take the spot where we showed Boskonf

**Comment by nheek** [comment]
I don't really know or remember where we showed Boskonf :((

**Comment by skivdal** [comment]
The IG text was really good, it works for me
I can translate it for the english version of the page

**Comment by nheek** [comment]
Perfect! When are we thinking of having it done? Like ideal timeframe

**Comment by skivdal** [comment]
Next couple of days ideally, don't feel like you have to push away other things for this though

**Comment by nheek** [comment]
![Skjermbilde 2024-02-16 kl  16 22 31](https://github.com/fribyte-code/fribyte.no/assets/83167470/a3437765-569a-4055-88ae-e30a5a672d70)
![Skjermbilde 2024-02-16 kl  16 22 37](https://github.com/fribyte-code/fribyte.no/assets/83167470/71a9c6c2-009a-4680-abeb-c790eca61fb8)

Thoughts for the first draft? I added some fireworks and made a banner component similar to the button one.

**Comment by nheek** [comment]
The links are pointed to their social media, since Sammen and Sammen Bergen has the same website (I assume) so I just pointed to the ig instead

**Comment by skivdal** [comment]
I like it! Maybe remove the sammen logo?
